### DISCUSSION TITLE
Preferred Featherwing Model?

### DISCUSSION THREAD

**Discussion by JoToSystems** [will127534/LoRa-GW-FeatherWing#5] [Q&A]
Hi, is there a particular model Featherwing that you have used?   From my understanding the FeatherWing family is about form factor and a standard connector with numerous processors of varying capability in the FeatherWing family.  Do you have any recommendation on which ones to use?

**Comment by will127534** [comment]
I've tested on SAMD21 but it should work on most processor family.
Just make sure the pins and communication (UART x1, SPI x1 with 2 chip enable, I2C x1) used by this featherwing is available with your processor choice. (If I remember correctly some of the processor doesn't have enough pins or UART for featherwing so they've NC some of them or shared the communication with USB to UART).
